### ISSUE TITLE
Prediction scores.

### ISSUE THREAD

**Issue by PrayushiFaldu** [dki-lab/Pangu#9]
How can we get prediction scores (or logits) along with predicted logical forms during inference?

**Comment by entslscheia** [comment]
In our current setup, we utilize the `predict` command from AllenNLP for inference. However, this method only produces predictions and doesn't provide metrics since it doesn't accept ground truth labels as input.

I think what you are looking for is the `evaluate` command (See the documentation [here](http://docs.allennlp.org/v0.9.0/api/allennlp.commands.evaluate.html)). However, the metrics logged in this way may not be the same as the metrics evaluated using the official script. As a result, we strongly recommend you to stick to the `predict` command to make predictions first and evaluate afterwards.
